kontrakter/dom/domsavgjoerelse/arbeidsversjon/domsavgjoerelse.schema.json

```diff
@@ -0,0 +1,990 @@
+        "saksnummer",
+        "straffesaksnummer",
+        "rettsmoeter",
+        "avgjoerelse"
+      ],
+      "type": "object",
+      "additionalProperties": false
+    },
+    "avgjoerelse": {
+      "properties": {
+        "avgjoerelseId": {
+          "$ref": "#/definitions/GUID",
+          "description": "Domstolen sin id for avgjørelsen. Kan bli brukt i forbindelse med anke/forlengelse. AvgjørelseId beholdes ved korrigeringer"
+        },
+        "avgjoerelseIdForrigeInstans": {
+          "$ref": "#/definitions/GUID",
+          "description": "Domstolen sin id til avgjørelsen i forrige instans. Settes kun dersom dette er en dom i en ankesak"
+        },
+        "avsagtDato": {
+          "$ref": "#/definitions/dato"
+        },
+        "kravId": {
+          "$ref": "#/definitions/GUID",
+          "description": "KravId i nåværende instans. Kan være null dersom saken er opprettet manuelt"
+        },
+        "opprinneligKravId": {
+          "$ref": "#/definitions/GUID",
+          "description": "KravId i første instans"
+        },
+        "avgjoerelseSiktedeTiltaltePersonerForetak": {
+          "properties": {
+            "personForetak": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/personForetakAvjoerelse"
+              },
+              "minItems": 1
+            }
+          }
+        }
+      },
+      "required": [
+        "avgjoerelseId",
+        "avsagtDato",
+        "opprinneligKravId",
+        "avgjoerelseSiktedeTiltaltePersonerForetak"
+      ],
+      "type": "object",
+      "additionalProperties": false
+    },
+    "personForetakAvjoerelse": {
+      "properties": {
+        "personForetak": {
+          "$ref": "#/definitions/personForetakRef",
+          "description": "Intern id til person eller foretak. Denne skal peke på en av personene/foretakene i 'siktedeTiltaltePersonerForetak' listen"
+        },
+        "resultat": {
+          "type": "array",
+          "description": "Resultat for hver basissak for den siktede/tiltalte",
+          "items": {
+            "$ref": "#/definitions/basissakResultat"
+          },
+          "minItems": 1
+        },
+        "reaksjoner": {
+          "type": "array",
+          "description": "Liste av reaksjoner for denne personen. Ved løslatelse vil denne listen være tom",
+          "items": {
+            "$ref": "#/definitions/reaksjonsType"
+          },
+          "minItems": 0
+        },
+        "erklaering": {
+          "$ref": "#/definitions/erklaering",
+          "description": "Erklæring fra siktede og påtale"
+        }
+      },
+      "required": ["personForetak", "resultat", "reaksjoner", "erklaering"],
+      "additionalProperties": false
+    },
+    "basissakResultat": {
+      "properties": {
+        "basissakId": {
+          "$ref": "#/definitions/internId",
+          "description": "Politiets interne id for basissaken"
+        },
+        "lovbudKombinert": {
```

Comment:
Hvis det er primær og subsidiært lovbud og den dømmes for det subsidiære så vil den subsidiære lovbudKombinertId stå her antar jeg.

Reply:
Skriv i description ?